tests/regressiontests/Makefile

```diff
@@ -373,6 +373,46 @@ test-simple: $(TESTDIR)/test-simple.sh
 	ERROR="require exactly 1" 	$(REVERSE) $<
 	$< pos -o 'uf ta' | grep -q 'OPT_S=uf ta,POS_S=pos,'
 	test -z "$(SHELLCHECK)" || $(SHELLCHECK) "$(TESTDIR)/test-simple.sh"
+test-simple: $(TESTDIR)/test-simple.sh
```

Comment:
Note to self - I have to refresh my memory how the test definitions work, this feels as unnecessary copy-pasting.